terraform/modules/lambda_s3/eb.tf

```
@@ -1,7 +1,7 @@
 resource "aws_cloudwatch_event_rule" "db_backup" {
   name                = "db-backup"
   description         = "Trigger lambda to perform db snapshot"
-  is_enabled          = true # update last
+  state               = true # update last
```

I think this wants to be

```suggestion
  state               = "ENABLED" # update last
```

to make it validate